packages/expo-modules-core/ios/Fabric/EXHostWrapper.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+#import <ExpoModulesCore/Platform.h>
+
```

Comment:
Make it clear that the initializer expects `RCTHost`. You can do a forward declaration here. It requires an import in the implementation file but you already have it there.

```suggestion

@class RCTHost;

```